crates/flipperzero/src/io.rs

```diff
@@ -39,6 +39,27 @@ pub enum Error {
     Uncategorized(sys::FS_Error),
 }
 
+impl ufmt::uDisplay for Error {
```

Comment:
The PR needs rebasing, and that will move it on top of 9ae08559ef2a7060cde78da7efb04e8ce758c91fwhich includes this impl.

Reply:
on it

Reply:
rebased